config/nginx.conf

```diff
@@ -0,0 +1,26 @@
+upstream gb-botler {
+  server unix:///var/www/gb-botler/shared/sockets/puma.sock;
+}
+
+server {
+  listen 8080;
```

Comment:
maybe this (and the guest port in the Vagrantfile) can go back to 80, with the $server_port fix? Can you try that?